docs/narr/cookiecutters.rst

```diff
@@ -3,10 +3,12 @@
 Pyramid cookiecutters
 =====================
 
-.. versionadded:: 1.8
+.. versionadded:: 1.10
 
 A :term:`cookiecutter` is a command-line utility that creates projects from :ref:`cookiecutters <cookiecutter:readme>` (project templates), e.g., creating a Python package project from a Python package project template.
 
+The unified cookiecutter ``pyramid-cookiecutter-theonecc`` merged features from the three previous cookiecutters which are now deprecated.
+
 The Pyramid cookiecutter has replaced the now deprecated Pyramid scaffolds, and should be used going forward and can be found here:
 
 * `pyramid-cookiecutter-theonecc <https://github.com/Pylons/pyramid-cookiecutter-theonecc>`_
```

Comment:
@stevepiercy 

I wasn't exactly sure what you wanted here when you said to apply the same language from `glossary.rst` -- can you verify this is good? Thanks!

Reply:
First, we have not been consistent with the use of `versionadded`, `versionchanged`, and `deprecated`. Sorry about that, we'll do better. Options include:

1. after relevant text, without an indented note
2. after relevant text, with an indented note
3. before relevant text without an indented note
4. before relevant text with indented note

Options 2 and 4 are preferred because they provide a clear context to the note or text. Options 1 and 3 may leave the context ambiguous (is this for the stuff above or below the directive?). Option 4 is best when we want to include multiple brief change notes as in [this example in the Sphinx docs](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#images).

With the decision to go with `-starter` as the one true CC, I'd go with option 4 in this case, and make sure that the text still makes sense.

* Keep `.. versionadded:: 1.8`, and append an indented note.

```rst
.. versionadded:: 1.8
    Added cookiecutter support.
```

* Add `.. versionchanged:: 1.10` for the -starter cookiecutter.

```
.. versionadded:: 1.10
    Merged features from ``pyramid-cookiecutter-alchemy`` and ``pyramid-cookiecutter-zodb`` into the single cookiecutter to rule them all, ``pyramid-cookiecutter-starter``.
```

* Add `.. deprecated:: 1.10` for -zodb and -alchemy, with a note.

```
.. deprecated:: 1.10
    Merged features from ``pyramid-cookiecutter-alchemy`` and ``pyramid-cookiecutter-zodb`` into the single cookiecutter to rule them all, ``pyramid-cookiecutter-starter``.
    ``pyramid-cookiecutter-alchemy`` and ``pyramid-cookiecutter-zodb`` are no longer supported.
    Use ``pyramid-cookiecutter-starter`` going forward.
```

Hopefully that gives you sufficient guidance to proceed.

Reply:
Awesome, thanks!